### ISSUE TITLE
Idea: email students when invited to join a group

### ISSUE THREAD

**Issue by brenfwd** [eecs-autograder/autograder.io#61]
Would be useful for emails to be sent to invitees to let them know they have been invited to join a project -- similar to GitHub project invites. Something like:

> Subject: Autograder.io Invite for [project name]
>
> [group owner] invited you to join their group for [project name] in [course name].
> [Click here to accept or reject this invitation](#)

**Comment by cycai0312** [comment]
> Would be useful for emails to be sent to invitees to let them know they have been invited to join a project -- similar to GitHub project invites. Something like:
> 
> > Subject: Autograder.io Invite for [project name]
> > [group owner] invited you to join their group for [project name] in [course name].
> > [Click here to accept or reject this invitation](#)
> 
> Would be willing to do a PR for this if desired :)

Yes! We are a pair of students in EECS 481 with an open-source contribution assignment. We plan to fix it in the next month. Thank you!

**Comment by james-perretta** [comment]
@brenfwd @cycai0312 Yes, I would accept a PR for this. Let's discuss the design & implementation details either here or over email (jameslp at umich). Thanks!

**Comment by james-perretta** [comment]
@brenfwd @cycai0312 Just a heads up that I'm about to merge https://github.com/eecs-autograder/ag-website-vue/pull/528, which switches the test runner from jest to vitest. You'll need to rebase, rerun `npm ci`, and make sure any tests you've written still work. Please email me if you run into any issues during that process.
